lib/galaxy/tool_util/deps/mulled/get_tests.py

```diff
@@ -259,6 +261,15 @@ def main_test_search(
     return {"container": container}
 
 
+def import_test_to_command_list(import_lang: str, import_: str) -> List[str]:
+    if import_lang == "python -c":
+        return ["python", "-c", f"import {import_}"]
+    elif import_lang == "perl -e":
+        return ["perl", "-e", f"use {import_}"]
+    else:
+        raise ValueError(f"Unsupported import_lang '{import_lang}'")
```

Comment:
Are you sure that's safe to do ? Seems like no exception would have been raised previously

Reply:
It's safe because we fill it with only these 2 values in `lib/galaxy/tool_util/deps/mulled/get_tests.py:get_commands_from_yaml()` .